docs/tutorials/analysis/3D/event_sampling.ipynb

```diff
@@ -747,8 +746,11 @@
    "outputs": [],
    "source": [
     "%%time\n",
+    "n_obs = len(tstarts)\n",
+    "irf_paths = [Path(irf_filename)] * n_obs\n",
```

Comment:
Is it useful here? It is only when reading that you need the path to the irf now, no?

Reply:
In general observations can have different irf files so the list of irfs_path have to be defined before the loop. I wanted to stress that in the notebook, even if it is not necessary for this use case (but we could use 2 different zenith angles for example).